string command
string parameters
---
int32 error_code
string response